gradle.properties

```diff
@@ -17,4 +17,4 @@ group=org.apache.kafka
 version=0.8.3-SNAPSHOT
 scalaVersion=2.10.5
 task=build
-org.gradle.jvmargs=-XX:MaxPermSize=512m -Xmx1024m -Xss2m
+org.gradle.jvmargs=-XX:MaxPermSize=512m -Xmx1024m -Xss2m -XX:-MaxFDLimit
```

Comment:
What is this change for? `MaxFDLimit` seems to be a deprecated flag (https://bugs.openjdk.java.net/browse/JDK-8010126?focusedCommentId=13315747&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13315747)